lib/types/applications.d.ts

```diff
@@ -464,3 +464,25 @@ export interface EditApplicationOptions {
     /** The tags for the application. Max 5 per application, 20 characters each. */
     tags?: Array<string>;
 }
+
+export interface RawApplicationEmoji extends Required<Omit<Emoji, "user" | "id">>  { id: string; user?: RawUser; }
+export interface ApplicationEmoji extends Omit<RawApplicationEmoji, "user" | "id" | "require_colons"> { id: string; requireColons?: boolean; user?: User; }
```

Comment:
put properties on their own line, and have ApplicationEmoji extend Emoji like it does above

Reply:
This is just a copy of https://github.com/OceanicJS/Oceanic/blob/080d6f738679506402db7f9add1d79fc445dd7e3/lib/types/guilds.d.ts#L136-L137

Reply:
I guess I'll leave it for now then